runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -221,6 +226,86 @@ private ClusterStatus getClusterHealth(Layout layout, Set<String> peerResponsive
         return peerResponsiveNodes;
     }
 
+    /**
+     * Fetches the initial layout to fetch the cluster status.
+     * This does not consume the corfu runtime layout as that can get stuck in an indefinite retry
+     * loop.
+     *
+     * @param layoutServers Layout Servers to fetch the layout from.
+     * @return Latest layout from the servers or null if none of them responded with a layout.
+     */
+    private Layout getClusterStatusLayout(List<String> layoutServers) {
+        Layout layout = null;
```

Comment:
Would be great if you add some logs here

Reply:
Done.